static/app/components/interactionStateLayer.tsx

```diff
@@ -71,7 +71,8 @@ const InteractionStateLayer = styled(
         `
       : ''}
 
-  *:disabled > && {
+  *:disabled &&,
+  *[aria-disabled="true"] && {
```

Comment:
Ok, now I'm starting to question everything, but why `&& {`? Is this some emotion stuff? If so, we should imo stop using it and stay closer to just regular CSS syntax

Reply:
`&` is a [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector) that in this case means "this component" (`InteractionStateLayer`). It's a common selector used in emotion, sass, less, etc. I'm using two of it, `&&`, to increase selector specificity, making this the highest-priority style block. It translates to `*:disabled .(class name of InteractionStateLayer).(class name of InteractionStateLayer)`